apps/meteor/app/slashcommands-topic/client/topic.ts

```diff
@@ -1,31 +1,32 @@
 import { Meteor } from 'meteor/meteor';
-import type { IMessage } from '@rocket.chat/core-typings';
 
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { ChatRoom } from '../../models/client/models/ChatRoom';
 import { callbacks } from '../../../lib/callbacks';
 import { hasPermission } from '../../authorization/client';
 import { handleError } from '../../../client/lib/utils/handleError';
 
-function Topic(_command: 'topic', params: string, item: IMessage): void {
-	if (Meteor.isClient && hasPermission('edit-room', item.rid)) {
-		Meteor.call('saveRoomSettings', item.rid, 'roomTopic', params, (err: Meteor.Error) => {
-			if (err) {
-				if (Meteor.isClient) {
-					handleError(err);
+slashCommands.add(
+	'topic',
+	function Topic(_command: 'topic', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.